anyscale/manifest.json

```
@@ -0,0 +1,43 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "e3d14556-99b3-4c17-a51a-4d66ef622601",
+  "app_id": "anyscale",
+  "display_on_public_website": false,
```

Also address the missing data fields for `auto_install` and `source_type_id`.

```suggestion
  "display_on_public_website": true,
```

---

The integration is not ready to be public yet.